inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/stream/TemplateField.java

```diff
@@ -0,0 +1,126 @@
+    @ExcelField(name = "Field name", validator = NonEmptyCellValidator.class, font = @Font(size = 16), style = @Style(bgColor = IndexedColors.LIGHT_TURQUOISE, width = 9000, allBorderColor = IndexedColors.BLUE, allBorderStyle = BorderStyle.THIN), headerFont = @Font(size = 20, color = IndexedColors.WHITE), headerStyle = @Style(bgColor = IndexedColors.DARK_BLUE, width = 9000, allBorderColor = IndexedColors.BLUE, allBorderStyle = BorderStyle.THIN))
+    @ApiModelProperty(value = "Field name", required = true)
+    private String fieldName;
+
+    @ExcelField(name = "Field type", validator = StreamFieldTypeCellValidator.class, font = @Font(size = 16), style = @Style(bgColor = IndexedColors.LIGHT_TURQUOISE, width = 6000, allBorderColor = IndexedColors.BLUE, allBorderStyle = BorderStyle.THIN), headerFont = @Font(size = 20, color = IndexedColors.WHITE), headerStyle = @Style(bgColor = IndexedColors.DARK_BLUE, width = 9000, allBorderColor = IndexedColors.BLUE, allBorderStyle = BorderStyle.THIN))
+    @ApiModelProperty(value = "Field type", required = true)
+    private String fieldType;
+
+    @ExcelField(name = "Field comment", font = @Font(size = 16), style = @Style(bgColor = IndexedColors.LIGHT_TURQUOISE, width = 10000, allBorderColor = IndexedColors.BLUE, allBorderStyle = BorderStyle.THIN), headerFont = @Font(size = 20, color = IndexedColors.WHITE), headerStyle = @Style(bgColor = IndexedColors.DARK_BLUE, width = 9000, allBorderColor = IndexedColors.BLUE, allBorderStyle = BorderStyle.THIN))
+    @ApiModelProperty("Field comment")
+    private String fieldComment;
+
+    @ApiModelProperty(value = "Is predefined field, 1: yes, 0: no")
+    private Integer isPredefinedField;
+
+    @ApiModelProperty(value = "Field value for constants")
+    private String fieldValue;
+
+    @ApiModelProperty(value = "Value expression of predefined field")
+    private String preExpression;
+
+    @ApiModelProperty("Is this field a meta field, 0: no, 1: yes")
+    @Builder.Default
+    private Integer isMetaField = 0;
+
+    @ApiModelProperty(value = "Meta field name")
+    private String metaFieldName;
+
+    @ApiModelProperty("Field format, including: MICROSECONDS, MILLISECONDS, SECONDS, SQL, ISO_8601"
+            + " and custom such as 'yyyy-MM-dd HH:mm:ss'. This is mainly used for time format")
+    private String fieldFormat;
+
+    @ApiModelProperty("Origin node name which stream fields belong")
+    private String originNodeName;
+
+    @ApiModelProperty("Origin field name before transform operation")
+    private String originFieldName;
+
+    @ApiModelProperty("Extra Param in JSON style")
+    private String extParams;
+
+    public TemplateField(int index, String fieldType, String fieldName, String fieldComment, String fieldValue) {
+        this.id = index;
+        this.fieldType = fieldType;
+        this.fieldName = fieldName;
+        this.fieldComment = fieldComment;
+        this.fieldValue = fieldValue;
+        // default
+        this.isMetaField = 0;
```

Comment:
`isMetaField` already has the default value 0, plz remove these two lines

Reply:
Fixed.